$NetBSD: patch-aa,v 1.8 2017/05/09 13:14:32 leot Exp $

Do not install the command line tools. The tools are provided separately by
mutagen-tools package.

--- setup.py.orig	2017-02-24 13:15:32.000000000 +0000
+++ setup.py
@@ -267,14 +267,14 @@ if __name__ == "__main__":
             "mutagen._senf",
             "mutagen._tools",
           ],
-          data_files=data_files,
-          scripts=[os.path.join("tools", name) for name in [
-            "mid3cp",
-            "mid3iconv",
-            "mid3v2",
-            "moggsplit",
-            "mutagen-inspect",
-            "mutagen-pony",
-          ]],
+#         data_files=data_files,
+#         scripts=[os.path.join("tools", name) for name in [
+#           "mid3cp",
+#           "mid3iconv",
+#           "mid3v2",
+#           "moggsplit",
+#           "mutagen-inspect",
+#           "mutagen-pony",
+#         ]],
           long_description=long_description,
     )
